apps/new-dealer/controllers/authority-holders-nationality.js

```diff
@@ -13,19 +13,19 @@ module.exports = class AuthorityHolderNationalityController extends BaseControll
       const thirdNationality = req.form.values[`${key}-third`].toLowerCase();
 
       if (firstNationality === secondNationality && firstNationality !== '') {
-        return new ErrorController(`${key}-second`, {
+        return new ValidationError(`${key}-second`, {
```

Comment:
I was just renaming variables because `ErrorController` is a *bad* variable name, but you are right and I will do that thing you said anyway.